src/test/loadExtension/loadExtension.test.ts

```diff
@@ -9,16 +9,16 @@ const cfFile: Uri = Uri.file(path.resolve(__dirname, '../../../src/test/compile/
 describe('Load extension', () => {
 
     beforeEach(async function() {
-        await commands.executeCommand('workbench.action.closeFolder');
+        await commands.executeCommand('workbench.action.closeActiveEditor');
```

Comment:
This line made the test case hang. Actually I cannot come up with a reason why this line should be there, but it also exists in the other test cases. When I remove this line, the test cases succeed. @sanderr @andraskvr Any idea why this line should be there.

Reply:
I don't know eiter.

Reply:
I think it might have been put there to make sure the tests start from a clean slate without an open editor, even if previous ones fail - but I'm not sure. If it's not necessary (anymore, maybe because of changes to vscode itself), we can remove it